tests/test_receivers.py

```diff
@@ -65,3 +67,30 @@ def test_build_cache_with_no_collections(app):
 
         # now the collection 'mycoll' should appear in `record0` of A!
         assert record0['_collections'] == ['mycoll']
+
+
+def test_cache_can_be_imported_from_a_config_variable(app):
+    InvenioCollections(app)
+    with app.app_context():
+        current_app.config['COLLECTIONS_CACHE'] = \
+            'werkzeug.contrib.cache:SimpleCache'
+
+        assert current_collections.cache is not None
+
+
+def test_cache_can_be_passed_from_a_config_variable(app):
+    InvenioCollections(app)
+    with app.app_context():
+        current_app.config['COLLECTIONS_CACHE'] = SimpleCache()
+
+        assert current_collections.cache is not None
+
+
+def test_cache_cannot_be_overridden_if_it_was_set(app):
+    first_cache = SimpleCache()
+    second_cache = SimpleCache()
+    InvenioCollections(app, cache=first_cache)
+    with app.app_context():
```

Comment:
We're talking about two different tests. My consideration on `current_collections` was about the first two tests, and the tests you told me to copy&paste was the third one, which fails for the stuff fixed in #71.